Adds the Guppy class runabout

kyres1 opened the pull request:
## About the PR
Adds a Guppy class runabout. It's the cheapest thing you can get in the CO now with very barebones amenities. It's zippy, maneuverable and completely useless at anything besides moving. The cost reflects that at a tiny 15100 credits.

Also this adds a manual for it, mostly a copy of the Sabine since they share common features for the most part.

## Why / Balance
There is otherwise nothing to purchase for simple transit. The next "smallest" choice would be the Sabine, which is a dedicated salvager with armament, or the Framework, which isn't even really a ship.

## Media
(The mapping-mode screenshot is the accurate one. Keen eyes would notice that it in fact has forward-facing thrusters and additional rear thrusters. That's the version in the PR)
![NVIDIA_Overlay_TReA5iiGV8](https://github.com/user-attachments/assets/2429c4fd-1168-4222-85b9-24a589793af9)

![NVIDIA_Overlay_8G8hWHqxqM](https://github.com/user-attachments/assets/cb5785ef-52b1-415d-a87b-9c0d26f6b954)

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X ] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [ X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added a small shuttle called the Guppy to the shipyard. It costs 15100 credits and is a good way to move around, and not much else.
-->

kyres1 committed:
guppyyyyyy

ghost commented:
![image](https://github.com/user-attachments/assets/57a543d9-2bde-48bf-b84c-2271659bc6b0)
pretty sure that console is in a window

kyres1 commented:
> dont put thrusters in walls. Also plastitanium for budget ship Lol!

I forgot there were alternatives! Walls are called plasteel where I'm from anyway!!!

Will fix both

kyres1 committed:
Tweaks for KiraZen and Starch

kyres1 commented:
Fixed @KiraZen-Underscore

Fixed @starch70 

The vent here was moved to be under the thruster instead on the far right side, and the wall was replaced. Otherwise this is up to date

Also added a fax machine and a pen and some paper
![NVIDIA_Overlay_OoexJfFptM](https://github.com/user-attachments/assets/9b07bebe-82df-45f9-9e6d-d6a227f768b4)

kyres1 committed:
resaving, adds a sink

kyres1 committed:
trying to fix the checks by removing changes

kyres1 commented:
fixing this PR would take more time than just making the shuttle again lmao

Will just open again later with a file that doesnt break (since I already merged a ship that works, somehow). I probably fudged the process up somewhere and made duplicate tiles or something. No way to tell with those very unhelpful check error descriptions (ignoring the yaml linter)
